index.html

```diff
@@ -38,6 +39,156 @@
     </div>
 
   </section>
+
+  <section class="work-section">
+    <div class="RWbox"><div class="RWork"><p>My Recent works</p>
+    <div class="workline"></div></div>
+    </div>
+    <div class="work-container">
+      <div class="workone"><img class="workphoto"src="/img/Img Placeholder.svg" alt="Recent work preview">
+       <p class="first-Rwork">Multi-Post Stories</p>
```

Comment:
- [ ] Kindly ensure that all your project titles (e.g., "Multi-Post Stories") use a header tag `hx`(h2 - h6) so as to enable screen readers and search engines to understand that you are describing each project title 👍🏽